Spigot-Server-Patches/0084-Configurable-Chunk-Inhabited-Time.patch

```diff
@@ -1,40 +1,47 @@
-From 7ac07ac07ac07ac07ac07ac07ac07ac07ac07ac0 Mon Sep 17 00:00:00 2001
+From abd072826676e2e451683a9fe5c4d8cee9f05399 Mon Sep 17 00:00:00 2001
 From: Aikar <aikar@aikar.co>
 Date: Mon, 28 Mar 2016 20:46:14 -0400
-Subject: [PATCH] Configurable Chunk Inhabited Timer
+Subject: [PATCH] Configurable Chunk Inhabited Time
 
 Vanilla stores how long a chunk has been active on a server, and dynamically scales some
 aspects of vanilla gameplay to this factor.
 
-For people who want all chunks to be treated equally, you can disable the timer.
+For people who want all chunks to be treated equally, you can chose a fixed value.
+
+This allows to fine-tune vanilla gameplay.
 
 diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
-index 7ac07ac07ac0..7ac07ac07ac0 100644
+index deb4ec25..08ac3696 100644
 --- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
 +++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
-@@ -233,4 +233,9 @@ public class PaperWorldConfig {
+@@ -233,4 +233,14 @@ public class PaperWorldConfig {
      private void firePhysicsEventForRedstone() {
          firePhysicsEventForRedstone = getBoolean("fire-physics-event-for-redstone", firePhysicsEventForRedstone);
      }
 +
-+    public boolean useInhabitedTime = true;
-+    private void useInhabitedTime() {
-+        useInhabitedTime = getBoolean("use-chunk-inhabited-timer", true);
++    public boolean useFixedInhabitedTime;
++    private void useFixedInhabitedTime() {
++        useFixedInhabitedTime = getBoolean("use-fixed-chunk-inhabited-time", false);
++    }
++
++    public int fixedInhabitedTime;
++    private void fixedInhabitedTime() {
++        fixedInhabitedTime = getInt("fixed-chunk-inhabited-time", 0);
```

Comment:
These should be a single block and share a section in the config.
Something similar to this.

```java
public boolean useFixedInhabitedTime;
public int fixedInhabitedTime;
private void fixedChunkInhabitedTime() {
    useFixedInhabitedTime = getBoolean("fixed-chunk-inhabited-time.enabled", false);
    fixedInhabitedTime = getInt("fixed-chunk-inhabited-time.value", 0);
}
```

We may want to consider migrating users of the old setting over as well. No reason to worry about that quite yet though.